ivy/functional/frontends/numpy/sorting_searching_counting/searching.py

```diff
@@ -92,3 +92,8 @@ def nanargmax(a, /, *, axis=None, out=None, keepdims=False):
 def nanargmin(a, /, *, axis=None, out=None, keepdims=False):
     a = _nanargminmax(a, axis=axis)
     return ivy.argmin(a, axis=axis, keepdims=keepdims, out=out)
+
+
+@to_ivy_arrays_and_back
+def extract(condition, arr):
+    return ivy.extract(condition, arr)
```

Comment:
Ivy doesn't have an extract function in its library. This function would have to be implemented as a combination of other Ivy functions